lib/finicity/v1/request/get_institution_by_id.rb

```diff
@@ -40,8 +40,7 @@ def url
           'aggregation/',
           'v1/',
           'institutions/',
-          "#{institution_id}/",
-          'details'
+          "#{institution_id}/"
```

Comment:
Based on documentation, this is also deprecated. Please see (https://community.finicity.com/s/article/Get-Institution) for the updated API.